modules/core/src/geometries/cube-geometry.js

```diff
@@ -50,18 +50,20 @@ const CUBE_TEX_COORDS = new Float32Array([
   0,  0,  1,  0,  1,  1,  0,  1
 ]);
 
+const ATTRIBUTES = {
+  POSITION: {size: 3, value: new Float32Array(CUBE_POSITIONS)},
+  NORMAL: {size: 3, value: new Float32Array(CUBE_NORMALS)},
+  TEXCOORD_0: {size: 2, value: new Float32Array(CUBE_TEX_COORDS)}
+};
+
 export default class CubeGeometry extends Geometry {
   constructor(props = {}) {
     const {id = uid('cube-geometry')} = props;
     super({
       ...props,
       id,
       indices: {size: 1, value: new Uint16Array(CUBE_INDICES)},
-      attributes: {
-        POSITION: {size: 3, value: new Float32Array(CUBE_POSITIONS)},
-        NORMAL: {size: 3, value: new Float32Array(CUBE_NORMALS)},
-        TEXCOORD_0: {size: 2, value: new Float32Array(CUBE_TEX_COORDS)}
-      }
+      attributes: {...ATTRIBUTES, ...props.attributes}
```

Comment:
Wondering if we might want this the other way? In what case would I want to use `CubeGeometry` and not use the e.g. the preset cube positions?

Reply:
> Wondering if we might want this the other way? In what case would I want to use CubeGeometry and not use the e.g. the preset cube positions?

I can see the argument for that. I tend to leave such choices to the users. If they want the original attributes, then they would just not override them, but if they really do want to change (e.g. normals) then they can? 

The attribute names generated by this geometry are well-documented so the behavior is predictable in that sense.